flight_declaration_operations/utils.py

```diff
@@ -1,136 +1,178 @@
+        priority: int,
+        state: str = "Accepted",
+    ) -> PartialCreateOperationalIntentReference:
+        all_v4d = self.convert_geo_json_to_volume4D(
+            geo_json_fc=geo_json_fc,
+            start_datetime=start_datetime,
+            end_datetime=end_datetime,
+        )
+
+        op_int_r = PartialCreateOperationalIntentReference(
+            volumes=all_v4d, state=state, priority=priority, off_nominal_volumes=[]
+        )
 
         return op_int_r
 
-    def convert_geo_json_to_volume4D(self, geo_json_fc: FeatureCollection, start_datetime: str, end_datetime:str) -> List[Volume4D]:
+    def convert_geo_json_to_volume4D(
+        self, geo_json_fc: FeatureCollection, start_datetime: str, end_datetime: str
+    ) -> List[Volume4D]:
         all_v4d = []
         # all_shapes = []
-        all_features = geo_json_fc['features']
+        all_features = geo_json_fc["features"]
         for feature in all_features:
-            geom = feature['geometry']
-            max_altitude = feature['properties']['max_altitude']['meters']
-            min_altitude = feature['properties']['min_altitude']['meters']
-            s = shape(geom)     
+            geom = feature["geometry"]
+            max_altitude = feature["properties"]["max_altitude"]["meters"]
+            min_altitude = feature["properties"]["min_altitude"]["meters"]
+            s = shape(geom)
             buffed_s = s.buffer(0.00001)
             # all_shapes.append(buffed_s)
-            self.all_features.append(buffed_s)  
+            self.all_features.append(buffed_s)
             # feature_union = unary_union(all_shapes)
-            # # TODO: build a better flightplan 
+            # # TODO: build a better flightplan
             # b = feature_union.minimum_rotated_rectangle
-            
+
             co_ordinates = list(zip(*buffed_s.exterior.coords.xy))
             # Convert bounds vertex list
             polygon_verticies = []
             for cur_co_ordinate in co_ordinates:
-                v = LatLngPoint(lat = cur_co_ordinate[1],lng = cur_co_ordinate[0])
+                v = LatLngPoint(lat=cur_co_ordinate[1], lng=cur_co_ordinate[0])
                 polygon_verticies.append(v)
 
             # remove the final point
             polygon_verticies.pop()
-                
-            volume3D = Volume3D(outline_polygon=Plgn(vertices= polygon_verticies),altitude_lower=Altitude(value=max_altitude,reference='W84',units='M'), altitude_upper=Altitude(value=min_altitude,reference='W84',units='M'))
 
-            volume4D = Volume4D(volume = volume3D, time_start=Time(format="RFC3339",value=start_datetime), time_end=Time(format="RFC3339", value=end_datetime))
+            volume3D = Volume3D(
+                outline_polygon=Plgn(vertices=polygon_verticies),
+                altitude_lower=Altitude(value=min_altitude, reference="W84", units="M"),
+                altitude_upper=Altitude(value=max_altitude, reference="W84", units="M"),
```

Comment:
Previously altitude_lower was set to max_altitude and altitude_upper was set to min_altitude